### ISSUE TITLE
Arrange items in status bar

### ISSUE THREAD

**Issue by wdanilo** [atom/status-bar#207]
Hi! Currenlty it is impossible for either user or anybody who builds something on atom (like we do) to arrange icons on the status bar. This implies a lot of pains:
  1. If there is a package like `platform-io-terminal` which registers itself on the very left side, the only way to move it on the right is to monkey patch it
  2. If I click inside Settings View and then inside text editor, the items on status bar change (for example the current language option appears). But it appears on right of busy signal, which jumps left and right about 300pxs when switching active tabs. It would be much more convinent to display optional information in a place that would not make other icons jump.
  3. I would like to re-arrange the items to different places by default, for example I want busy signal to be on the very left of status bar. Currently only monkey patching everything is possible, which is a little ugly solution. 

I've seen that some progress was done 2 years ago to this topic: https://discuss.atom.io/t/announce-move-status-items/18451 , but it seems its not maintained.

**Comment by rgawenda** [comment]
Yes, it seems that move-status-items isn't actively maintained, but I'm using it on 1.21beta without tinkering.

I would love to see it integrated as an integrated configurable feature of status-bar.

**Comment by Zakini** [comment]
+1 for this. My status bar is gradually getting more cluttered. I'd like to stay on top of it!

**Comment by rzjnzk** [comment]
+1

**Comment by LeptoSpira** [comment]
Can this please be done? It has been about 2 years since this was asked, it seems no one has taken action to add this yet.

**Comment by norton64** [comment]
As a warning to anyone on the dev team under 35...  As one gets older and needs stronger glasses, one ends up increasing the font-size to the point where the items in the status bar crash into each other and become unreadable.  For all of us over-35 users, can this enhancement be re-prioritized?
